packages/components/src/theme/ComposerTheme/Components/Radio.ts

```diff
@@ -0,0 +1,71 @@
+import { radioAnatomy as parts } from '@chakra-ui/anatomy';
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme';
+import { PartsStyleInterpolation } from '@chakra-ui/theme-tools';
+
+const sizes: Record<string, PartsStyleInterpolation<typeof parts>> = {
+  sm: ({ theme }) => ({
+    ...theme.textStyles['text-body-meta-regular'],
+  }),
+  md: ({ theme }) => ({
+    ...theme.textStyles['text-body-regular'],
+  }),
+  lg: ({ theme }) => ({
+    ...theme.textStyles['text-body-large-regular'],
+  }),
+};
+
+export const Radio: ComponentMultiStyleConfig = {
+  parts: parts.keys,
+  baseStyle: {
+    control: {
+      color: 'text-light',
+      borderColor: 'ui-element-outline-default',
+      _focus: {
+        _invalid: {
+          borderColor: 'alert-error-default',
+        },
+        _hover: {
+          bg: 'primary-hover',
+          borderColor: 'primary-hover',
+          _invalid: {
+            bg: 'alert-error-default',
+          },
+        },
+      },
+
+      _checked: {
+        borderColor: 'primary-default',
+        bg: 'primary-default',
+        color: 'text-light',
```

Comment:
Is this needed? Isn't it enough to have it defined in the color of the control part?

Reply:
it is needed because a color value is defined in `Checkbox` and I need to override it